BParkingNano/test/validate_pr.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/env bash
+
+set -o errexit
+set -o nounset
+
+: ${CMSSW_BASE:?"CMSSW_BASE is not set!  Run cmsenv!"}
+
+PRID=$1
+
+git pull origin master
```

Comment:
maybe add the option to specify the remote label, if someone does not call it "origin"?

Reply:
OK, done